client/cmd/playground/window/model.go

```diff
@@ -89,130 +115,229 @@ func (m *model) View() string {
 }
 
 func (m *model) generateWindowResultView() string {
-	buff := &bytes.Buffer{}
+	buff := new(bytes.Buffer)
 	table := tablewriter.NewWriter(buff)
-	table.SetHeader([]string{"Version", "Start Time", "End Time"})
-	table.Append(m.generateWindowTableRowView(1))
-	table.Append(m.generateWindowTableRowView(2)) //nolint: gomnd
+
+	interval, err := m.calculateInterval()
+	if err != nil {
+		table.SetHeader([]string{"ERROR"})
+		table.Append([]string{err.Error()})
+	} else {
+		startRow := interval.Start().Format(time.RFC3339)
+		endRow := interval.End().Format(time.RFC3339)
+
+		table.SetHeader([]string{"Start Time", "End Time"})
+		table.Append([]string{startRow, endRow})
+	}
+
 	table.Render()
 	return buff.String()
 }
 
 func (m *model) generateWindowInputHintView() string {
 	var hint string
 	switch m.currentCursor {
-	case pointToTruncateTo:
+	case pointToSizeInput:
+		hint = "empty or positive numeric value only\n"
+	case pointToSizeUnit:
 		hint = `valid values are:
-- M: truncate to month
-- w: truncate to week
-- d: truncate to day
-- h: truncate to hour
+- None or <empty>: only valid if size input is empty
+- h: hour
+- d: day
+- w: week
+- M: month
+- y: year
 
 press (shift+up) or (shift+w) to increment value
 press (shift+down) or (shift+s) to decrement value
 `
-	case pointToOffset:
-		hint = `valid formats are:
-- nMmh: example 1M2h meaning 1 month 2 hours
-- mh: example 2h meaning 2 hours
+	case pointToDelayInput:
+		hint = "empty or numeric value only (negative is allowed)"
+	case pointToDelayUnit:
+		hint = `valid values are:
+- None or <empty>: only valid if delay input is empty
+- h: hour
+- d: day
+- w: week
+- M: month
+- y: year
 
-n can be negative, while m can be negative only if nM does NOT exist
+press (shift+up) or (shift+w) to increment value
+press (shift+down) or (shift+s) to decrement value
 `
-	case pointToSize:
-		hint = `valid formats are:
-- nMmh: example 1M2h meaning 1 month 2 hours
-- mh: example 2h meaning 2 hours
+	case pointToTruncateToUnit:
+		hint = `valid values are:
+- None or <empty>: truncation happens based on the unit if size
```

Comment:
`if` is typo?

Reply:
updated, with small update for better readability